.github/workflows/ci.yaml

```diff
@@ -0,0 +1,192 @@
+name: OHCL Build and Validation
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    tags:
+      - '**'
```

Comment:
I initially put that but left it back to the wildcard as a "just in case" measure (though we might create tags for other purposes later on, so it might be best to be slightly more restrictive), I was also thinking of using `rolling-lts/hcl-*/*` too.